postgres/src/main/scala/package.scala

```diff
@@ -0,0 +1,127 @@
+package anorm
+
+import java.util.UUID
+
+import scala.util.control.NonFatal
+
+import play.api.libs.json.{
+  JsError, JsObject, Json, JsSuccess, JsValue, Reads, Writes }
+
+import org.postgresql.util.PGobject
+
+package object postgresql extends PGJson {
```

Comment:
That's already the case for UUID, for Json I d prefer not to have such dependency in the core. I generic JSON module could be.